src/OrderBundle/Entity/OrderManager.php

```
@@ -64,4 +66,44 @@ public function getOrder($orderId)
 
         return $qb->getQuery()->getOneOrNullResult();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = array())
+    {
+        $query = $this->getRepository()
+            ->createQueryBuilder('o')
+            ->select('o');
+
+        $fields = $this->getEntityManager()->getClassMetadata($this->class)->getFieldNames();
+        foreach ($sort as $field => $direction) {
+            if (!in_array($field, $fields)) {
+                unset($sort[$field]);
```

its better to throw an exception, when field is wrong